README.md

```diff
@@ -189,9 +259,9 @@ To get up and running with this project:
     dbt run
     ```
 
-    > **NOTE:** If this steps fails, it might mean that you need to make small changes to the SQL in the models folder to adjust for the flavor of SQL of your target database. Definitely consider this if you are using a community-contributed adapter.
+    > **NOTE:** If you decide to run this project in your own data warehouse (outside of this DuckDB demo) and this steps fails, it might mean that you need to make small changes to the SQL in the models folder to adjust for the flavor of SQL of your target database. Definitely consider this if you are using a community-contributed adapter.
```

Comment:
```suggestion
    > **NOTE:** If you decide to run this project in your own data warehouse (outside of this DuckDB demo) and steps fail, it might mean that you need to make small changes to the SQL in the models folder to adjust for the flavor of SQL of your target database. Definitely consider this if you are using a community-contributed adapter.
```

Reply:
After you commit this suggestion, it's ready to merge!

Reply:
@dbeatty10 FYI on this PR to clean up the README